galaxy_ng/tests/unit/api/rh_auth.py

```diff
@@ -15,7 +15,8 @@ def user_x_rh_identity(username, account_number=None):
                 {"username": "%(username)s",
                 "email": "%(username)s",
                 "first_name": "%(title_username)s",
-                "last_name": "%(title_username)sington"
+                "last_name": "%(title_username)sington",
+                "is_org_admin": true
```

Comment:
do the unit tests in `test_api_ui_synclists.py` check for `is_org_admin` being true and/or false?, seems most test logic changes are log outputs rather than asserts

Reply:
If the unit tests are run in 'insights' mode, the viewset permissions use the insights access_policy which will enforce is_org_admin. 

In 'standalone' mode, most of those tests are skipped.  And by default the tests run in 'standalone' mode (in CI as well), so most of those test don't run. 

(There is a wider issue related to how/when/when settings and access_policy get loaded that some of the deployment mode specific code can't be fully mocked or overridden. Running full test suite in 'insights' mode causes auth related tests to fail for the same reason except vice versa).

So at the moment, most of test_api_ui_synclist tests are only hit when running only the synclists tests using 'insights' deployment mode. Fixing the deployment mode dependent tests (in synclists and elsewhere) needs a deeper fix.